System/Process/Internals.hs

```diff
@@ -414,6 +429,8 @@ foreign import ccall unsafe "runInteractiveProcess"
         -> Ptr FD
         -> Ptr FD
         -> Ptr FD
+        -> Ptr CGid
```

Comment:
It looks like using `-1` is probably ok, but I'm still skeptical since [setgid's](http://pubs.opengroup.org/onlinepubs/009695399/functions/setgid.html) and [setuid's](http://pubs.opengroup.org/onlinepubs/009695399/functions/setuid.html) documentation don't mention it. I'll revert to the pointer approach since I think it's the safer option.